### PR TITLE
add pre-commit to Pipfile

### PR DESCRIPTION
**Pull request by rawxfish**
Adds pre-commit to Pipfile, so that the stuff that Github runs when we do pull request runs first locally when commit is created.

### CODE REVIEW

**Comment by erwa** [comment]
For this to work, you need to run `pre-commit install` manually first in order for the git hook scripts to get installed in `.git/hooks/pre-commit`, right? (per https://pre-commit.com/#3-install-the-git-hook-scripts)

Could we update the dev-guide.md to mention this?

**Comment by erwa** [comment]
I like this pre-commit hook. Thanks for adding it!

**Comment by rawxfish** [comment]
I think this is already in the dev guide: https://github.com/cardinalitypuzzles/cardboard/blob/master/dev-guide.md#set-up-pre-commit-checks

File: Pipfile [hunk 1]
```diff
@@ -18,6 +18,7 @@ celery = "*"
 django-celery-results = "*"
 redis = "*"
 "discord.py" = "*"
+pre-commit = "*"
 
 [dev-packages]
 pre-commit = "*"
```
Comment (rgossiaux) [inline Pipfile:24]: Isn't this already in the dev-packages section below? I think if you run `pipenv install --dev` it should be installed.

Comment (rawxfish) [reply Pipfile:24]: oh you're right, I forgot the --dev, thanks!

**Pull request closed without merging**